packages/components/src/Combobox/Combobox.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ReactElement, ReactNode } from "react";
+import { useAssert } from "@jobber/hooks/useAssert";
+import { ComboboxContent } from "./components/ComboboxContent";
+import { ComboboxAction } from "./components/ComboboxAction";
+import { ComboboxContextProvider } from "./ComboboxProvider";
+import {
+  ComboboxTriggerButton,
+  ComboboxTriggerChip,
+} from "./components/ComboboxTrigger";
+
+export interface ComboboxProps {
+  readonly children: ReactElement | ReactElement[];
+}
+
+export const COMBOBOX_TRIGGER_COUNT_ERROR_MESSAGE =
```

Comment:
Naive new person question: Do constants normally live with the root component, or is there ever a constants.ts file to house them?

Reply:
This is another good topic to bring up with the whole team because it seems to be a changing pattern...or it depends.

The only `constants.ts` files we have are related to `components-native` components.
I just noticed that `DataList` has a `DataList.const.ts` file that houses a lot of constants.
Older components seem to have the constants live within the root component.

Reply:
exactly what Taylor said

in mobile, where I've been living for the past year ish, we do `constants.ts` but I didn't see many/any in Atlantis. despite the fact that I did the opposite here, I do like separating them into their own file especially when they are user facing messages for translations